src/constants/about.js

```diff
@@ -104,3 +104,5 @@ export const DEVICES_DATA = [
     videoSrc: susiDesktop,
   },
 ];
+
+export const shareUrl = 'https://susi.ai';
```

Comment:
Do not make a new variable. Use variable from https://github.com/fossasia/susi.ai/blob/b0b666be89192b478dc69875607421cb67870279/src/utils/urls.js#L13

Reply:
Done Sir!